libkbfs/folder_branch_ops.go

```diff
@@ -92,8 +92,8 @@ type fboMutexLevel mutexLevel
 
 const (
 	fboMDWriter fboMutexLevel = 1
-	fboHead                   = 2
-	fboBlock                  = 3
+	fboHead     fboMutexLevel = 2
```

Comment:
Why?  I thought the previous way was idiomatic.  (Same with the `MetadataVer` stuff above.)